packages/superset-ui-plugin-chart-table/src/renderer.tsx

```diff
@@ -132,7 +134,7 @@ export const getRenderer = ({
     >
       <Parent>
         {column.format ? (
-          column.format(value)
+          column.format.format(value as any)
```

Comment:
@kristw  this one I'm not so sure. I got type error if I don't add `.format`

Reply:
That's fine. There was some typing edge case that typescript fail to infer.